### PR TITLE
A number of documentation fixes

### PR DESCRIPTION
**Pull request by flimzy**
Grammar, spelling, typos, formatting, etc.

### CODE REVIEW

**Comment by kxepal** [comment]
Thanks for the fixes, but it seems build is broken by these changes. Please check travis logs for the issues.

**Comment by flimzy** [comment]
It's passing now.

File: src/couchapp/ddocs.rst [hunk 1]
```diff
@@ -93,23 +92,23 @@ Reduce and rereduce functions
...
 
     [query_server_config]
     reduce_limit = false
 
-While disabling ``reduce_limit`` might be useful for debug proposes, remember,
-that main task of reduce functions is to *reduce* mapped result, not to make it
-even bigger. Generally, your reduce function should converge rapidly to a single
+While disabling ``reduce_limit`` might be useful for debug proposes, remember
+that the main task of reduce functions is to *reduce* mapped result, not to make
+them bigger. Generally, your reduce function should converge rapidly to a single
```
Comment (kxepal) [inline src/couchapp/ddocs.rst]: Why did you replace `it` with `them`? `it` originally was referenced to `result` which is singular while `them`  now cannot do the same as it is plural.

Comment (flimzy) [inline src/couchapp/ddocs.rst]: It should probably say "to reduce mapped results" (which is probably what I read), or "to reduce the mapped result", in which case the singular "it" would be correct.

Comment (kxepal) [inline src/couchapp/ddocs.rst]: Feel free to improve wording if you think it could be clear (:

Comment (flimzy) [inline src/couchapp/ddocs.rst]: It's fixed now (to my satisfaction, anyway... feel free to disagree) :)

Comment (kxepal) [inline src/couchapp/ddocs.rst]: LGFM, thanks! (:

**Comment by kxepal** [comment]
Merged, thanks a lot! (:

**Pull request merged**